notebooks/official/automl/automl_image_object_detection_online_prediction.ipynb

```diff
@@ -147,285 +156,318 @@
+        "import sys\n",
         "\n",
-        "# Set the project id\n",
-        "! gcloud config set project {PROJECT_ID}"
+        "if \"google.colab\" in sys.modules:\n",
+        "\n",
+        "    from google.colab import auth\n",
+        "\n",
+        "    auth.authenticate_user()"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "region"
+        "id": "4f872cd812d0"
       },
       "source": [
-        "#### Region\n",
+        "### Set Google Cloud project information and initialize Vertex AI SDK for Python\n",
         "\n",
-        "You can also change the `REGION` variable used by Vertex AI. Learn more about [Vertex AI regions](https://cloud.google.com/vertex-ai/docs/general/locations)."
+        "To get started using Vertex AI, you must have an existing Google Cloud project and [enable the Vertex AI API](https://console.cloud.google.com/flows/enableapi?apiid=aiplatform.googleapis.com). Learn more about [setting up a project and a development environment](https://cloud.google.com/vertex-ai/docs/start/cloud-environment)."
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "region"
+        "id": "set_project_id"
       },
       "outputs": [],
       "source": [
-        "REGION = \"us-central1\"  # @param {type: \"string\"}"
+        "PROJECT_ID = \"[your-project-id]\"  # @param {type:\"string\"}\n",
+        "\n",
+        "# Set the project id\n",
+        "! gcloud config set project {PROJECT_ID}\n",
+        "\n",
+        "LOCATION = \"us-central1\"  # @param {type: \"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "gcp_authenticate"
+        "id": "bucket:mbsdk"
       },
       "source": [
-        "### Authenticate your Google Cloud account\n",
+        "### Create a Cloud Storage bucket\n",
         "\n",
-        "Depending on your Jupyter environment, you may have to manually authenticate. Follow the relevant instructions below."
+        "Create a storage bucket to store intermediate artifacts such as datasets."
       ]
     },
     {
-      "cell_type": "markdown",
+      "cell_type": "code",
+      "execution_count": null,
       "metadata": {
-        "id": "FvQeFm3Gv5mR"
+        "id": "bucket"
       },
+      "outputs": [],
       "source": [
-        "**1. Vertex AI Workbench**\n",
-        "* Do nothing as you are already authenticated."
+        "BUCKET_URI = f\"gs://your-bucket-name-{PROJECT_ID}-unique\"  # @param {type:\"string\"}"
       ]
     },
     {
       "cell_type": "markdown",
       "metadata": {
-        "id": "ad1138a125ea"
+        "id": "create_bucket"
       },
       "source": [
-        "**2. Local JupyterLab instance, uncomment and run:**"
+        "**Only if your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."
```

Comment:
Remove "Only"":
- "**If your bucket doesn't already exist**: Run the following cell to create your Cloud Storage bucket."